src/AvtaleSide/GodkjenningSteg/Oppsummering/Avtaleparter/Avtaleparter.less

```diff
@@ -1,10 +1,63 @@
+@mobile-breakpoint: 576px;
```

Comment:
Brukes @media (min-width: @screen-sm-min) ellers i prosjektet.
Bør vel bruke samme bredde overalt for å avgjøre mobil/desktop.

Reply:
vel skal være 2 kolonner på oppsummeringen. Foruten når en er i mobilview